pkg/generators/models/testdata/cases/parameter_model/api.yaml

```diff
@@ -0,0 +1,40 @@
+openapi: 3.0.0
+info:
+  version: 0.1.0
+  title: Test
+
+paths:
+  "/foo/{id}":
+    parameters:
+      - name: param1
+        in: query
+        schema:
+          type: string
+          format: uuid
+      - name: id
+        required: true
+        in: path
+        schema:
+          type: string
+          format: uuid
```

Comment:
Can we please add a test case where parameters are referenced from the components? This where the old generator fails.